src/data/graphics/trainers.h

```diff
@@ -526,23 +526,23 @@ const struct SpriteFrameImage gTrainerBackPicTable_Steven[] =
 // .backPic goes functionally unused, since none of these pics are compressed
 // and the place they would get extracted to gets overwritten later anyway
 // the casts are so they'll play nice with the strict struct definition
-#define TRAINER_BACK_SPRITE(trainerPic, yOffset, sprite, pal, anim)                                                                                       \
-    [TRAINER_BACK_PIC_##trainerPic] =                                                                                                                     \
-    {                                                                                                                                                     \
-        .coordinates = {.size = 8, .y_offset = yOffset},                                                                                                  \
-        .backPic = {(const u32 *)gTrainerBackPic_##sprite, TRAINER_PIC_SIZE * ARRAY_COUNT(gTrainerBackPicTable_##sprite), TRAINER_BACK_PIC_##trainerPic}, \
-        .palette = {gTrainer##pal, TRAINER_BACK_PIC_##trainerPic},                                                                                        \
-        .animation = sBackAnims_##anim,                                                                                                                   \
+#define TRAINER_BACK_SPRITE(trainerPic, yOffset, sprite, pal, anim)                                                                    \
+    [trainerPic] =                                                                                                                     \
+    {                                                                                                                                  \
+        .coordinates = {.size = 8, .y_offset = yOffset},                                                                               \
+        .backPic = {(const u32 *)gTrainerBackPic_##sprite, TRAINER_PIC_SIZE * ARRAY_COUNT(gTrainerBackPicTable_##sprite), trainerPic}, \
```

Comment:
This is probably up for discussion as well, but as above, I would have all the global variables outside of their macro for the sake of searchability. The fact eg `trainerPic` as argument pops up multiple means the existence of this macro is still justified, but it needs to be changed slightly.

Reply:
Done